lib/tests/basic-call.rb

```diff
@@ -32,6 +32,11 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
+def sip_to_tel(uri)
+  uri =~ /sip:(\d+)@.+/
```

Comment:
This won't work with + or NP info. This isn't a problem for the tests, but worth a comment